tests/utils/many_sorters_test/many_sorters.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/pingcap/tiflow/cdc/sorter"
 	"github.com/pingcap/tiflow/cdc/sorter/unified"
 	"github.com/pingcap/tiflow/pkg/config"
+	cerrors "github.com/pingcap/tiflow/pkg/errors"
```

Comment:
Is this file(test caes) dead code, it is not reasonable to exist missing import. 🤔